oio/common/wsgi.py

```diff
@@ -109,6 +111,10 @@ def dispatch_request(self, req):
             resp = getattr(self, 'on_' + endpoint)(req)
         except HTTPException as exc:
             resp = exc
+        except ServiceBusy as exc:
+            if self.logger:
+                self.logger.error('ServiceBusy: %s', exc.message)
+            resp = ServiceUnavailable(escape(exc.message))
```

Comment:
This has already been fixed in another branch: https://github.com/open-io/oio-sds/commit/b1eb7691b0eb7e9be1c7b2ac320aeedb8530d23e